rest_framework/renderers.py

```diff
@@ -64,15 +70,16 @@ class JSONRenderer(BaseRenderer):
     # We don't set a charset because JSON is a binary encoding,
     # that can be encoded as utf-8, utf-16 or utf-32.
     # See: http://www.ietf.org/rfc/rfc4627.txt
-    # Also: http://lucumr.pocoo.org/2013/7/19/application-mimetypes-and-encodings/
+    # http://lucumr.pocoo.org/2013/7/19/application-mimetypes-and-encodings/
```

Comment:
We shouldn't have to alter our docstring text in order to keep the linter happy. I'm happy to have strict linting for almost everything, but I think the line length should be a guideline rather than mandatory, because of cases like this where it forces authors to alter what they would otherwise write.